dashboard/db/migrate/20170921234349_add_storage_id_to_channel_tokens.rb

```diff
@@ -0,0 +1,6 @@
+class AddStorageIdToChannelTokens < ActiveRecord::Migration[5.0]
+  def change
+    add_column :channel_tokens, :storage_id, :integer
+    add_index :channel_tokens, :storage_id
```

Comment:
Can you include the query changes to validate the index?

Reply:
The query changes are going to be that eventually both here
https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/channel_token.rb#L34
and here
https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/channel_token.rb#L43

We search using storage_id instead of user_id.